contrib/alloc_instrumentation.py

```diff
@@ -41,10 +41,10 @@ def print_stacks(stack_count, sort_by_count):
 
     sort_dict = counts if sort_by_count else sizes
     ordered_list = [(val, backtrace) for (backtrace, val) in sort_dict.items()]
-    ordered_list.sort(reverse=True)
+    ordered_list.sort()
```

Comment:
We are reversing the sorting order; any reason to do it?

Reply:
This list is printed repeatedly to standard out, largest values on the bottom is far more readable.